testing/gdm/gdm.initd

```diff
@@ -1,18 +1,21 @@
 #!/sbin/openrc-run
 
-GDM_LOG_FILE="/var/log/${SVCNAME}.log"
+supervisor=supervise-daemon
 
-description="GNOME display manager"
-command=/usr/sbin/${SVCNAME}
-command_args="${gdm_opts}"
-command_background="true"
-start_stop_daemon_args="-w 100 --stdout $GDM_LOG_FILE --stderr $GDM_LOG_FILE"
-pidfile="/run/${SVCNAME}.pid"
+name="Display Manager"
+description="GNOME Display Manager"
 
-start_pre() {
-        checkpath -f -m 0644 -o ${SVCNAME}:${SVCNAME} "$GDM_LOG_FILE"
-}
+command=/usr/sbin/gdm
+command_args="$gdm_opts"
+
+GDM_LOG_FILE="/var/log/${RC_SVCNAME}.log"
+
+supervise_daemon_args="-w 100 --stdout $GDM_LOG_FILE --stderr $GDM_LOG_FILE"
```

Comment:
-w doesn't seem to be an argument for supervise-daemon and neither is stdout or stderr, but we can get rid of that anyway, gdm logs everything to /var/log/gdm/gdm.log